plugins/quickinspector/quickinspector.cpp

```diff
@@ -308,12 +308,11 @@ void RenderModeRequest::apply()
         const QByteArray mode = renderModeToString(RenderModeRequest::mode);
         QQuickWindowPrivate *winPriv = QQuickWindowPrivate::get(window);
 #if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
-        QObject::connect(window.get(), &QQuickWindow::beforeSynchronizing, this, [this, winPriv, mode](){
+        QObject::connect(window.get(), &QQuickWindow::beforeSynchronizing, this, [this, winPriv, mode]() {
```

Comment:
squash this commit bot patch locally please (and consider setting up pre-commit locally)

Reply:
Done.